homeassistant/components/blink/sensor.py

```diff
@@ -36,7 +32,6 @@
     SensorEntityDescription(
         key=TYPE_WIFI_STRENGTH,
         translation_key="wifi_rssi",
-        native_unit_of_measurement=SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
```

Comment:
Removing the UoM causes the device class to fail as it requires either the decibels or decibels mw.

Reply:
I tested this and it looked "ok" on the UI.  The units are not "right" with the new API, it's no longer dBm rather just 0-5 value.

Reply:
Do I need to change the device_class? What should it be?

Reply:
Remove it, Probably if you open the logs you will see some warnings about it

Reply:
I don't see any errors in the log... but it still wasn't "correct".   Should be good now.